internal/proto/milvus.proto

```diff
@@ -228,6 +228,19 @@ message DescribeIndexResponse {
   repeated IndexDescription index_descriptions = 2;
 }
 
+message GetIndexBuildProgressRequest {
+  common.MsgBase base = 1;
+  string db_name = 2 ;
+  string collection_name = 3;
+  string field_name = 4;
+  string index_name = 5;
+}
+
+message GetIndexBuildProgressResponse {
+  common.Status status = 1;
+  float progress = 2;
```

Comment:
Should contain two int values, which represent the total number of entites ​​and the number of indexed entites respectively